Circuit/Simulation/Simulation.cs

```diff
@@ -3,11 +3,12 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Numerics;
 using System.Reflection;
 using Util;
 using LinqExpr = System.Linq.Expressions.Expression;
-using LinqExprs = System.Linq.Expressions;
 using ParamExpr = System.Linq.Expressions.ParameterExpression;
+using System.Numerics;
```

Comment:
Minor nit: this is used twice (line 6 and here).

Reply:
Fixed